styles/Home.module.scss

```diff
@@ -5,3 +5,84 @@
 .MainArea {
   @apply bg-color-1 flex-grow;
 }
+.Modal {
+  @apply w-11/12 max-w-2xl outline-none overflow-hidden;
+  @apply bg-opacity-0;
+}
+
+.modalImageContainer {
+  @apply absolute w-52 h-72 z-40;
```

Comment:
Can you set the `w` or the `h` we can discuss this more if you want but the css `aspect-ratio` property is so much better here set it to match the aspect ratio of the image. Also how does this work when the page goes below this width?

Edit: I actually dont think this file was meant to change.